src/Round.js

```diff
@@ -0,0 +1,39 @@
+class Round {
+  constructor(array) {
+    this.turns = 0
+    this.deck = array
+    this.incorrectGuesses = []
+    this.currentCard = ''
+    this.percentCorrect = 0
+  }
+  returnCurrentCard() {
+    return this.deck.cards[this.turns]
+  }
+  calculatePercentCorrect() {
+    if (this.turns > 0) {
+      let numWrong = this.incorrectGuesses.length
+      let numRight = this.turns - numWrong
+      this.percentCorrect = (numRight / this.turns) * 100
+      return this.percentCorrect;
+    }
+    return this.percentCorrect;
+  }
+  takeTurn(guess) {
```

Comment:
Could probably use a refactor and a second look, but getting to "functional" was a bit tight in the first place. Excited to give it a second glance.